core/src/test/java/io/questdb/griffin/engine/groupby/SampleByTest.java

```diff
@@ -156,6 +156,39 @@ public void testGeohashInterpolated() throws Exception {
         );
     }
 
+    @Test
+    public void testSampleByAlignedToCalendarWithTimezoneEndingWithSemicolon() throws Exception {
+        assertQuery("k\tcount\n" +
+                        "1970-01-03T00:00:00.000000Z\t6\n" +
+                        "1970-01-03T06:00:00.000000Z\t6\n" +
+                        "1970-01-03T12:00:00.000000Z\t6\n" +
+                        "1970-01-03T18:00:00.000000Z\t2\n",
```

Comment:
Both tests would be a tiny bit more readable if "create table" was put in next line .

Reply:
agree and done!